terraform/environments/bootstrap/single-sign-on/main.tf

```diff
@@ -53,6 +53,13 @@ data "aws_ssoadmin_permission_set" "migration" {
   name         = "modernisation-platform-migration"
 }
 
+data "aws_ssoadmin_permission_set" "database-mgmt" {
+  provider = aws.sso-management
+
+  instance_arn = local.sso_instance_arn
+  name         = "modernisation-platform-database-mgmt"
```

Comment:
The SSO plan is failing as the name is too long, maybe rename to mp-instance-management ?